build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [ ! -d "build" ]; then
+    mkdir build
+fi
+cd build && cmake .. && make $*
```

Comment:
there probably ought to be an additional condition to bring the user back to the originating directory.  Otherwise it can be surprising to the developer.